modules/dxp/apps/ai-hub/ai-hub-site-initializer/src/main/java/com/liferay/ai/hub/site/initializer/internal/AIHubSiteInitializer.java

```diff
@@ -246,6 +366,18 @@ private void _deployWorkflowDefinitions(long groupId) throws Exception {
 			"This is the text to be shortened: {{text}}");
 	}
 
+	private String _escape(String json) {
+		for (String[] strings : _JSON_ESCAPE_STRINGS) {
+			json = StringUtil.replace(json, strings[0], strings[1]);
+		}
+
+		return json;
+	}
+
+	private static final String[][] _JSON_ESCAPE_STRINGS = {
+		{"\\", "\\\\"}, {"\"", "\\\""}
+	};
```

Comment:
@MrJohn1911 define this as a local variable in the method `_escape` because it is only used there